enterprise/plugins/com.liferay.ide.scripting.core/src/com/liferay/ide/scripting/core/GroovyScriptProxy.java

```diff
@@ -68,8 +68,8 @@ public Object invoke( Object proxy, Method method, Object[] args ) throws Throwa
         }
         catch( Throwable t )
         {
-            ScriptingCore.logError( "Error in script method " + method.getName(), t );
-            error = t;
+//            ScriptingCore.logError( "Error in script method " + method.getName(), t );
```

Comment:
This should not be fixed here (which would disable all error logging, but should be disabled at a higher level (in the editor itself)

Reply:
Thanks, will resend